packages/react-instantsearch-core/src/connectors/__tests__/connectInfiniteHits.js

```diff
@@ -5,39 +5,35 @@ jest.mock('../../core/createConnector', () => x => x);
 describe('connectInfiniteHits', () => {
   describe('single index', () => {
     const createSingleIndexContext = () => ({
-      context: {
-        ais: {
-          mainTargetedIndex: 'index',
-        },
-      },
+      mainTargetedIndex: 'index',
```

Comment:
We removed the function in the other tests we should stay consistent, either we keep it or we remove it.

Reply:
the reason I kept it here is because the object gets modified in these tests, but not in others